docs/sources/enterprise/usage-insights/dashboard-datasource-insights.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Dashboard and data source insights"
+description = "Understand how your dashboards and data sources are used"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/dashboard-datasource-insights.md"]
+weight = 200
++++
+
+# Dashboard and data source insights
+
+For every dashboard and data source, you can access usage information in one click.
+
+## Dashboard insights
+
+> **Note:** Available in Grafana Enterprise v7.0+.
+
+You can see dashboard usage information by clicking on the `Dashboard insights` button in the top bar.
+
+{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_button.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+It shows two kinds of information:
+
+- **Stats:** Shows the daily query count and error count for the last 30 days.
```

Comment:
```suggestion
- **Stats:** Count the number of daily queries, and errors, for the past 30 days.
```

Reply:
Is this daily errors or one month’s worth of errors?

Reply:
@oddlittlebird Wondering about our style here, because this is not actually where the counting is happening (I think). What is your take on imperative versus not?

Reply:
Same goes for line that follows.

Reply:
It's a daily number of errors, you can see what is displayed to the users here: https://grafana.com/static/img/docs/enterprise/dashboard_insights_stats.png
I made an update but not exactly the one you suggested.